### PR TITLE
Appended to docu-string for function make_array and updated function …

### PR DESCRIPTION
**Pull request by stevenworks**
[ ] Wrote test for feature
[ ] Added changes to CHANGELOG.md
[ ] Bumped version number (delete if unneeded)

**Changes proposed:**
Added args and return type to docu-string for function make_array. Also updated function sample_proportions to use multinomial method of Generator class per Numpy recommendations: https://numpy.org/doc/stable/reference/random/generated/numpy.random.multinomial.html

### CODE REVIEW

**Review by adnanhemani — CHANGES_REQUESTED** [review]
Overall, looks good - thanks for these changes! Please revise as per the comments and then it should be good to merge. Thanks!

File: datascience/util.py [hunk 1]
```diff
@@ -16,14 +16,19 @@
 import collections
 
 # Change matplotlib formatting. TODO incorporate into a style?
-plt.rcParams["patch.force_edgecolor"] = True
+plt.rcParams['patch.force_edgecolor'] = True
```
[the remark this answers is not in the document]
Comment (stevenworks) [reply datascience/util.py:19]: The use of single quotes maintains consistency with the rest of the codebase where single quotes are used. Honestly, maybe not necessary but might as well.

File: datascience/util.py [hunk 2]
```diff
@@ -254,4 +260,4 @@ def is_non_string_iterable(value):
         return False
     if hasattr(value, '__iter__'):
         return True
-    return False
```
[the remark this answers is not in the document]
Comment (stevenworks) [reply datascience/util.py:257]: I apologize for missing that.

File: datascience/util.py [hunk 3]
```diff
@@ -84,7 +89,7 @@ def plot_normal_cdf(rbound=None, lbound=None, mean=0, sd=1):
 
         ``mean`` (numeric): mean/expectation of normal distribution
 
-        ``sd`` (numeric): standard deviation of normal distribution
+        ``sd`` (numeric): standard deviation of normal distribution
```
[the remark this answers is not in the document]
Comment (stevenworks) [reply datascience/util.py:92]: I missed this minor error as well.

File: datascience/util.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (adnanhemani) [reply datascience/util.py:19]: Fair enough. Let's leave your changes then :)

File: datascience/util.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (adnanhemani) [reply datascience/util.py:257]: Thanks for this change :)

**Review by adnanhemani — APPROVED** [review]
Thanks for making the requested changes - a quality PR! :D

**Pull request merged**